src/Menu/Menu.js

```diff
@@ -396,10 +400,27 @@ class Menu extends Component {
         event.preventDefault();
         this.decrementKeyboardFocusIndex();
         break;
+      default:
+        const hotKeys = this.hotKeyHolder.append(key);
+        if (this.setFocusIndexStartsWith(hotKeys)) {
+          event.preventDefault();
+        }
     }
     this.props.onKeyDown(event);
   };
 
+  setFocusIndexStartsWith(keys) {
+    const found = React.Children.toArray(this.props.children).findIndex((child) => {
+      const {primaryText} = child.props;
+      return primaryText && new RegExp(`^${keys}`, 'i').test(primaryText);
```

Comment:
`primaryText` is supposed to be a `node`. Shouldn't we make sure it's a `string`?